docs/spec/tendermint-fork-cases/MC_n4_f3.tla

```diff
@@ -4,15 +4,11 @@ CONSTANT Proposer \* the proposer function from 0..NRounds to 1..N
 \* the variables declared in TendermintAcc3
 VARIABLES
   round, step, decision, lockedValue, lockedRound, validValue, validRound,
-  msgsPropose, msgsPrevote, msgsPrecommit, evidence
-
-\* an operator for type annotations
-a <: b == a
+  msgsPropose, msgsPrevote, msgsPrecommit, evidence, action
 
 INSTANCE TendermintAccDebug3 WITH
   Corr <- {"c1"},
-  Defective <- {"a2", "a3"},
-  Byzantine <- {"f4"},
+  Faulty <- {"f2", "f3", "f4"},
```

Comment:
shouldn't this be:

```suggestion
  Faulty <- {"a2", "a3", "f4"},
```

?

Reply:
We have realized that it does help to distinguish between Byzantine and amnesic validators. They are all faulty. The property `Accountability` tells us that we can detect `f+1` faulty validators if `Agreement` is violated.